test/set_schema_test.lua

```diff
@@ -1327,9 +1327,14 @@ g.test_gh_108_fieldno_index_no_space_format = function()
     t.assert_equals(err, nil)
 
     t.assert_equals(box.space['weird_space']:format(), {})
-    t.assert_equals(
-        box.space['weird_space'].index['pk'].parts,
-        {{fieldno = 1, type = 'unsigned', is_nullable = false}})
+
+    local res = box.space['weird_space'].index['pk'].parts
+    if res[1].exclude_null == nil then
+        res[1].exclude_null = false
+    end
+    t.assert_equals(res,
```

Comment:
I prefer `assert_items_include` for branching results. It seems like a hack, but still better than overwriting test response.

```lua
tarantool> t = require('luatest')
---
...

tarantool> variant_1 = {{fieldno = 1, type = 'unsigned', is_nullable = false,
          exclude_null = false}}
---
...

tarantool> variant_2 = {{fieldno = 1, type = 'unsigned', is_nullable = false,
          exclude_null = nil}}
---
...

tarantool> variant_3 = {{fieldno = 1, type = 'unsigned', is_nullable = false,
          exclude_null = true}}
---
...

tarantool> allowed_variants = {variant_1, variant_2}
---
...


tarantool> t.assert_items_include(allowed_variants, {variant_1})
---
...

tarantool> t.assert_items_include(allowed_variants, {variant_2})
---
...

tarantool> t.assert_items_include(allowed_variants, {variant_3})
---
- error:
    status: fail
    class: LuatestError
    message: "Expected all item values from: \n{{{exclude_null = true, fieldno = 1,
      is_nullable = false, type = \"unsigned\"}}}\nTo be present in: \n{\n    {{exclude_null
      = false, fieldno = 1, is_nullable = false, type = \"unsigned\"}},\n    {{fieldno
      = 1, is_nullable = false, type = \"unsigned\"}},\n}"
...
```

Reply:
Thanks for the hint, done.